tests/layer_tests/tensorflow_tests/test_tf_ApproximateEqual.py

```diff
@@ -0,0 +1,42 @@
+import numpy as np
+import tensorflow as tf
+import pytest
+from common.tf_layer_test_class import CommonTFLayerTest
+
+class TestApproximateEqual(CommonTFLayerTest):
+    def _prepare_input(self, inputs_info):
+        assert 'tensor1:0' in inputs_info
+        assert 'tensor2:0' in inputs_info
+        tensor1_shape = inputs_info['tensor1:0']
+        tensor2_shape = inputs_info['tensor2:0']
+        inputs_data = {}
+        inputs_data['tensor1:0'] = np.random.uniform(-10, 10, tensor1_shape).astype(np.float32)
+        inputs_data['tensor2:0'] = np.random.uniform(-10, 10, tensor2_shape).astype(np.float32)
```

Comment:
please use `rng = np.random.default_rng()` and `rng.random` generator. See other layer tests having it. That is because `np.random.uniform` is being deprecated and we need to avoid to use it